src/components/SearchTracePage/TraceSearchForm.js

```diff
@@ -317,7 +317,6 @@ const mapDispatchToProps = dispatch => {
         tag: tagsToQuery(tags) || undefined,
         minDuration: minDuration || null,
         maxDuration: maxDuration || null,
-        traceID: traceIDsToQuery(traceIDs) || undefined,
```

Comment:
shouldn't this remain? When the search is performed by external URL that contains trace IDs in it, we definitely want them to be used, but when the search is performed via the search form in the UI we don't want the IDs from the previous search to be used

Reply:
@yurishkuro No, this doesn't need to be here. This is only taking them out of the form submission.

The LOC that handles traceIDs from the URL: [src/components/SearchTracePage/index.js#L72](https://github.com/uber/jaeger-ui/blob/master/src/components/SearchTracePage/index.js#L72)

Reply:
+1